packages/app/ui/socialLinks.tsx

```diff
@@ -1,59 +1,44 @@
-import { View } from "app/design-system";
-import { A } from "dripsy";
-import Icon from "@expo/vector-icons/MaterialCommunityIcons";
```

Comment:
can we remove this package from the dependencies?

Reply:
Still used by dashboard tabbar

Reply:
could you replace it there?